src/components/Pages/AboutMe/Aboutme.jsx

```diff
@@ -29,8 +30,27 @@ const Aboutme = () => {
             <Button>Download CV</Button>
           </div>
         </div>
-      </div>
+      </Div>
     </>
   );
 };
+const Div = styled.div`
+  .about-img-holder img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .section-description-about {
+    font-size: calc(13px + (15 - 13) * ((100vw - 300px) / (1300 - 300)));
```

Comment:
Why do we need to calculate the font size? Try using `em`

Reply:
Done